src/views/MainPage/MainPage.js

```diff
@@ -65,21 +76,17 @@ function renderMainView() {
   const creditsButton = Button('Credits', 'credits-button', true, 'click', handleNavigationButtonClick);
 
   navContainer.append(quizButton, leaderboardButton, adoptionButton, creditsButton);
-  const navItems = navContainer.querySelectorAll('a');
-
-  navItems.forEach((item) => {
-    item.addEventListener('click', () => {
-      alert(`Hello! I'm ${item.dataset.name} site`);
-    });
-  });
 }
 
-async function renderAnimalFact() {
-  const factItem = document.querySelector('.bubble-text');
+async function renderAnimalFact(factObj) {
+  const factSentenceItem = document.querySelector('.bubble.fact .bubble-text');
   const factSentence = document.createElement('p');
-  const FACT =
-    "Cat's eyes shine in the dark because of the tapetum, a reflective layer in the eye, which acts like a mirror.";
 
-  factSentence.innerText = FACT;
-  factItem.append(factSentence);
+  const fact = await retrieveAnimalFact();
+
+  factObj.sentence = fact;
```

Comment:
extract it to the separate function and use it in the main - in renderAnimalFact() we should keep code responsible for just rendering

Reply:
done